### DISCUSSION TITLE
Title screen removal

### DISCUSSION THREAD

**Discussion by AkiraArtuhaxis** [AlexisL61/JackboxPatcherFR#314] [Ideas]
Instead of having a title screen with three buttons, it'd be better if everything was integrated into the launcher itself. The patcher and settings buttons would become small buttons you can click on each corner of the launcher.
![OneScreenConcept](https://github.com/AlexisL61/JackboxUtility/assets/93792993/842270ca-e872-409e-ba3b-9a5b607b2d9d)
The only thing that'd be removed would be the little notifications/banners. While these are cool, these...don't really have much use, so they'd be removed.

**Comment by MeDustyy** [comment]
My concept:
![Знімок екрана 2023-08-07 1948051](https://github.com/AlexisL61/JackboxUtility/assets/58863601/d27f2ffe-6f2c-4f6f-ac7f-da992fdbd4cf)

But I don't agree that notifications/banners should be removed, as an option the "Server News" tab can be created for them or something like that.

**Reply by AkiraArtuhaxis** [reply]
That actually looks pretty clean.
About the news, the main problem is where to put them. Currently, they work because they're little banners which incentivises the user to click on them to know what they're about. If there was a "News" tab like the three buttons in your concept, there wouldn't be much incentive to click them. Also, the English server...pretty much would never have news of any kind. This is indeed a complicated topic, and will need to be discussed a bit more.
